internal/core/command_interceptor.go

```diff
@@ -0,0 +1,42 @@
+package core
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func CombineInterceptor(interceptors ...CommandInterceptor) CommandInterceptor {
+	var result CommandInterceptor
```

Comment:
Rename it `finalInterceptor` or something else but the first time I read your code I thought it was the result of the run.